src/scss/patterns/_dropdown.scss

```diff
@@ -21,11 +21,11 @@
 [data-dropdown] {
   @include tools.z-index('dropdown');
   background: var(--bg);
-  border: thin solid var(--border-fade);
+  border: thin solid var(--active);
   box-shadow: 0 0 var(--quarter-shim) var(--shadow);
-  max-width: 14em;
+  max-width: calc(100vw - var(--page-margin) * 2);
```

Comment:
curious how we can up with this equation?

Reply:
the viewport-width, minus the two (left and right) page margins, should be the full content width.

Reply:
ah, so that is for small screens. Got it.